heroku_connect/contrib/health_check.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import subprocess
+
+from health_check.backends import BaseHealthCheckBackend
+from health_check.exceptions import ServiceReturnedUnexpectedResult
+
+from ..conf import settings
+
+logger = logging.getLogger('heroku-health-check')
+
+
+class HerokuConnectHealthCheck(BaseHealthCheckBackend):
+    def check_status(self):
+        run_args = ['heroku', 'connect:info']
+        heroku_app = settings.HEROKU_CONNECT_APP_NAME
+        if not heroku_app:
+            raise ServiceUnavailable('App name is a required argument')
+        run_args.extend(['-a', heroku_app])
+        try:
+            output = subprocess.check_output(run_args)
+        except subprocess.SubprocessError as e:
+            raise ServiceReturnedUnexpectedResult(e)
+        else:
+            return self.get_status_from_heroku_output(output.decode('utf-8'))
+
+    def get_status_from_heroku_output(self, output):
+        """Parse status to fetch Connection and sync Status.
+
+        Sample output:
```

Comment:
https://devcenter.heroku.com/articles/heroku-connect-api#step-8-monitor-the-connection-and-mapping-status